src/runtime/virtcontainers/sandbox.go

```diff
@@ -2235,6 +2237,29 @@ func (s *Sandbox) AddDevice(ctx context.Context, info config.DeviceInfo) (api.De
 	return add, nil
 }
 
+// GetVfioDeviceGuestPciPath return a device's guest PCI path by its host BDF
+func (s *Sandbox) GetVfioDeviceGuestPciPath(hostBDF string) types.PciPath {
```

Comment:
Seems like an odd place to put this function here. Consider adding this to https://github.com/kata-containers/kata-containers/blob/main/src/runtime/pkg/device/drivers/vfio.go

Reply:
[src/runtime/pkg/device/drivers/vfio.go](https://github.com/kata-containers/kata-containers/blob/main/src/runtime/pkg/device/drivers/vfio.go) is for a single VFIODevice. This function needs to iterate all `VFIODev`s **via `Sandbox.devManager`** to find the `VFIODev` which matches `hostBDF`, IMO it's not reasonable to move this function to vfio.go.